articles/extensions/custom-extensions.md

```diff
@@ -8,35 +8,109 @@ To begin creating an your custom extension, please feel free to fork/clone any o
 - [Auth0 Extension Boilerplate](https://github.com/auth0/auth0-extension-boilerplate)
 - [Auth0 Extension with API Boilerplate](https://github.com/auth0/auth0-extension-boilerplate-with-api)
 - [Auth0 Extension with React Boilerplate](https://github.com/auth0/auth0-extension-boilerplate-with-react)
+- [Auth0 Extension with Hooks](https://github.com/auth0/auth0-extension-boilerplate-hooks)
 
-Alternatively, you may following the Development Instructions provided via the "New Extension" window that appears when you click on the "+ Extension" button. This allows you to create your own extension using the command line.
+Alternatively, you may following the _Development Instructions_ provided via the _New Extension_ window that appears when you click on the _+ CREATE EXTENSION_ button. To view the _Development Instructions_, navigate to the [Extensions](${uiURL}/#/extensions) page of the [Management Portal](${uiURL}). Click on the _+ CREATE EXTENSION_ button. On the popup displayed click on the _Check out this command line tool_ link. The _Development Instructions_ popup is displayed. These instructions allow you to create your own extension using the command line.
+
+> Learn more about the `wt-cli` by visiting the [github repository](https://github.com/auth0/wt-cli) and the [documentation](https://webtask.io/docs/wt-cli).
 
 ## Installing a Custom Extension
-Once you have created your own extension, you may install it manually via the Auth0 Management Portal.
+Once you have created your own extension, you may install it manually via the [Extensions](${uiURL}/#/extensions) page of the [Auth0 Management Portal](${uiURL}).
+
+Near the top right-hand side of the window, click the _+ CREATE EXTENSION_ button.
+
+![](/media/articles/extensions/custom/create-extension.png)
 
-Near the top right-hand side of the window, click the "+ NEW EXTENSION" button. In the "New Extension" window that pops open, provide the GitHub URL to the repository that contains the files required by your extension.
+In the _New Extension_ window that pops open, provide the GitHub URL to the repository that contains the files required by your extension.
+
+![](/media/articles/extensions/custom/new-extension.png)
 
 > At this time, only repositories hosted by GitHub may be used.
 
 Alternatively, you may host your files elsewhere and simply provide a link to the `webtask.json` file in the box (e.g. `http://example.com/webtask.json`).
 
-![]()
+Once you have provided the link to your files and clicked _Continue_, you will be prompted to install the extension. If you would like to proceed, click _Install_.
 
-Once you have provided the link to your files and clicked "Create", you will be prompted to install the extension. If you would like to proceed, click "Install."
+![](/media/articles/extensions/custom/install-custom-ext.png)
 
-At this point, you have two options in addition to completing the installation of the extension. You may:
+## Configuring the Installed Extension
```

Comment:
We have to remove this section. There is no configuration required after installing a custom extension.